src/containerapp/azext_containerapp/_params.py

```diff
@@ -241,6 +248,7 @@ def load_arguments(self, _):
         c.argument('server', help="The container registry server, e.g. myregistry.azurecr.io")
         c.argument('username', help='The username of the registry. If using Azure Container Registry, we will try to infer the credentials if not supplied')
         c.argument('password', help='The password of the registry. If using Azure Container Registry, we will try to infer the credentials if not supplied')
+        c.argument('identity', help="The managed identity with which to authenticate to the Azure Container Registry (instead of username/password). Use 'system' for a system-defined identity or a resource id for a user-defined identity. The managed identity should have been assigned acrpull permissions on the ACR before deployment (use 'az role assignment create --role acrpull ...').")
```

Comment:
Please refer to document [managed_identity_command_guideline](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) to design parameters related to managed identity

Reply:
Thanks for linking the guidelines, but I don't think they guidelines are relevant to this parameter. In this case, the `--identity` parameter is for using MSI so the Container App can authenticate with an Azure Container Registry (with an app that already exists). So we aren't creating managed identities at resource creation. Also, we already have a series of `az containerapp identity assign/remove/show` commands

Reply:
Is this also the scenario of assign identity? If so, could we refer to the `--system-assigned` and `--user-assigned <AzureResourceId1> <AzureResourceId2>` parameters used by the `az <resource> identity assign` command?
In addition, may it need to assign multiple identities in the future?

Reply:
I'm not sure that would make sense in this case -- we're simply letting users pass in a managed identity (just one) instead of an ACR username/password to access the ACR from the container app

Reply:
assigning identities to a containerapp is a separate action and already covered by the separate command `az containerapp identity assign`

Reply:
Got it, thanks~